components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,301 @@
+all the **postgresql** instances and ensures to choose a leader in case of leader database failure. For **elasticsearch**, there is a **msae**
+concept of leader follower. In case of any databse failure, a leader election occurs, and a new leader is chosen.
+
+## Disaster Recovery (DR)
+
+Disaster Recovery (DR) is a comprehensive plan for the recovery of critical operations and systems after catastrophic events. High
+availability is focused on serious but more typical failures, such as a failing component or server. A DR plan may cope with the loss of an entire region, for example, although both are related.
+
+DR cluster can be implemented on Chef Automate through a regular backup and restore measures syncing the data from the production cluster to the
+DR cluster. Typically, these two clusters are located in different data centers or cloud provider regions. This is intended to provide an environment that can be made production in a short period with minimal data loss.
+
+# System & Software Requirements 
+
+## For platform support 
+
++----------------------+-----------------------------------------------+
+|      Operating       |                               Tested          |
+| Systems              | Versions                                      |
++======================+===============================================+
+| Red Hat Enterprise   |  7, 8 (For 8 or above                         |
+| Linux                | versions, **SELinux** configuration must be   |
+|                      | permissive. By default, in RHEL               |
+| (64 Bit OS)          | 8, **SELinux** configuration is enforced).    |
+|                      |                                               |
+|                      |                                               |
+|                      |                                               |
+|                      | Red Hat Enterprise Linux derivatives include  |
+|                      | Amazon Linux v1 (using RHEL 6 packages) and   |
+|                      | v2 (using RHEL 7 packages).                   |
++----------------------+-----------------------------------------------+
+| Ubuntu (64 Bit OS)   |  14.04.x, 16.04.x, 18.04.x, 20.04.x           |
++----------------------+-----------------------------------------------+
+| Centos (64 Bit OS)   |  7                                            |
++----------------------+-----------------------------------------------+
+
+## Virtual Machine (VM) instances Type 
+
++--------------+-------------+--------------------------+--------------+
+|              |       Type  |                    RAM   |              |
+| Instance     |             |                          | Volume-size  |
++==============+=============+==========================+==============+
+| PostgreSQL   | t3a.medium  | 4GB RAM For Test and 8G  | 50GB         |
```

Comment:
@jayvikramsharma1 : are we using the t3a.medium ?

Reply:
yes, in a2ha.rb it was t3a.medium, but as we had discussed we will make it t3.medium instead of t3a.mediumm as it have high clock speed

### T3

The T3 instances feature either the 1st or 2nd generation Intel Xeon Platinum 8000 series processor (Skylake-SP or Cascade Lake) with a sustained all core Turbo CPU clock speed of up to 3.1 GHz. Additionally there is support for the new Intel Advanced Vector Extensions 512 (AVX-512) instruction set, offering up to 2x the FLOPS per core compared to the previous generation T2 instances.

### T3a

The T3a instances feature AMD EPYC 7000 series processors with an all core turbo clock speed of 2.5 GHz. The AMD-based instances provide additional options for customers that do not fully utilize the compute resources and can benefit from a cost savings of 10%.